bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/EvoCommunicator.java

```diff
@@ -314,26 +326,30 @@ private void initializeEpromData() {
         private int tcpPort = 10000;
         private String pcPassword = "0000";
 
+        private boolean useEncryption;
+
         EvoCommunicatorBuilder(PanelType panelType) {
             this.panelType = panelType;
         }
 
         @Override
         public IParadoxCommunicator build() {
-            if (panelType != PanelType.EVO48 && panelType != PanelType.EVO96 && panelType != PanelType.EVO192) {

```

Comment:
Why is this check not needed anymore?

Reply:
Because I've extracted this check outside in the ParadoxBuilderFactory and this check is duplicating the other one. 
There is a switch/case in ParadoxBuilderFactory which has default clause that will throw ParadoxRuntimeException if panel type is not supported.